### PR TITLE
Issue 3627 - New ticket UI

### LINKED ISSUE #3627: Custom tickets - Tickets details - New ticket ui
### Description
Right now we have a new ticket functionality but it lacks nice design. We need to apply the ui to make it pretty

### Goals
- [ ] As a user I want to be able to create tickets using a nice design 

### Tasks
- [ ] Apply the design  from figma

### Related Resources
[Design](https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?node-id=789%3A5282)

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #3627 

#### Description
Added dropdown menu to create a new ticket from template
Added button to cancel new ticket creation
Done some code cleaning for the the ActionMenu component

### CODE REVIEW

File: frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx [hunk 1]
```diff
@@ -69,12 +68,13 @@ export const TicketsList = ({ tickets }: TicketsListProps) => {
 		<>
 			<Filters>
 				{getTemplatesForFilter().map(({ name, _id }) => (
-					<FilterChip
+					<TemplateName
```
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx:71]: The filter chips in the ticket list card had a broken styling, so I just copied the previous style from the original branch and re-applied it here

File: frontend/src/v5/ui/controls/actionMenu/actionMenu.component.tsx [hunk 2]
```diff
@@ -40,27 +40,28 @@ const applyCloseMenuToActionMenuItems = (el: any, handleClose: () => void) => {
...
-		onClick: handleClick,
+	const TriggerButton = cloneElement(TriggerButtonInput as ReactElement, {
+		onClick: (e) => {
+			TriggerButtonInput.props.onClick?.(e);
+			handleClick(e);
+		},
 	});
 
-	const MenuChildren = menuChildren.map((child) => applyCloseMenuToActionMenuItems(child, handleClose));
+	const MenuChildren = Children.toArray(children).map((child) => applyCloseMenuToActionMenuItems(child, handleClose));
```
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/controls/actionMenu/actionMenu.component.tsx:64]: Yes, that's exactly it

File: frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (The-Daniel) [reply frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx:71]: The only broken styling I can see if the border colour and perhaps some spacing issues. It's probably best to keep the FilterChip component as this is what it was made for it just hasn't been reviewed by George yet

[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx:71]: Ahhhhhh, my bad!

File: frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsListCard.component.tsx [hunk 3]
```diff
@@ -25,19 +25,43 @@ import { useParams } from 'react-router-dom';
...
-import { Button } from '@controls/button';
 import { CardContext } from '@components/viewer/cards/cardContext.component';
+import AddIcon from '@assets/icons/add_circle.svg';
+import { sortBy } from 'lodash';
 import { TicketsList } from './ticketsList.component';
+import { ActionMenu, MenuItem, NewTicketButton } from './ticketsList.styles';
 import { TicketsCardViews } from '../tickets.constants';
 import { ViewerParams } from '../../../routes.constants';
 
+const NewTicketMenu = () => {
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsListCard.component.tsx:36]: Can you put this one in a separte file?

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

**Pull request merged**